packages/augur-ui/src/modules/market-cards/containers/market-card.ts

```diff
@@ -0,0 +1,27 @@
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+
+import MarketCard from "modules/market-cards/market-card";
+import { toggleFavorite } from "modules/markets/actions/update-favorites";
+
+const mapStateToProps = state => ({
+  isLogged: state.authStatus.isLogged,
+  isMobile: state.appStatus.isMobile,
+  pendingLiquidityOrders: state.pendingLiquidityOrders,
+  currentAugurTimestamp: state.blockchain.currentAugurTimestamp,
+  reportingWindowStatsEndTime: state.reportingWindowStats.endTime,
```

Comment:
let's go with this for now. For markets in fast resolution, we'll need to get markets dispute window to get the end time. middleware will give us that information on market info. We'll update then